clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapChainMap.java

```diff
@@ -214,6 +214,15 @@ public void put(K key, Chain chain) {
     }
   }
 
+  void remove(K key) {
+    heads.writeLock().lock();
+    try {
+      heads.remove(key);
```

Comment:
This should either be `heads.removeNoReturn(key)` or `heads.remove(key).close()`. I'd prefer the former since the later is slightly broken although it shows no symptoms at the moment.

Reply:
Probably good to add a unit test to ChainMapTest here at the same time.